Packs/Core/Integrations/CoreIOCs/CoreIOCs_test.py

```diff
@@ -70,7 +70,6 @@ def test_http_request_bad_json(self, requests_mock):
         assert e.value.message == f'Could not parse json out of {text}'
         assert e.value.res.status_code == 200
         assert isinstance(e.value.exception, json.JSONDecodeError)
-        assert e.value.exception.args == ('Expecting value', 'not a json')
```

Comment:
Can we replace it with another check instead of removing?

Reply:
the behavior is tested in line 70 (the message includes `not a json` message)